src/frontend/Komponenter/Personoversikt/LagBehandlingModal.tsx

```diff
@@ -0,0 +1,193 @@
+
+            axiosRequest<Ressurs<void>, null>({
+                method: 'POST',
+                url: `/familie-ef-sak/api/tilbakekreving/fagsak/${fagsakId}/opprett-tilbakekreving`,
+            })
+                .then((response) => {
+                    if (response.status === RessursStatus.SUKSESS) {
+                        settHentTilbakekrevinger(true);
+                        settVisModal(false);
+                        settToast(EToast.TILBAKEKREVING_OPPRETTET);
+                    } else {
+                        settFeilmeldingModal(response.frontendFeilmelding || response.melding);
+                    }
+                })
+                .finally(() => {
+                    settSenderInnBehandling(false);
+                });
+        }
+
+        if (valgtBehandlingstype === Behandlingstype.REVURDERING) {
+            if (kanStarteRevurdering()) {
+                settSenderInnBehandling(true);
+                const revurderingInnhold: RevurderingInnhold = {
+                    fagsakId: fagsakId,
+                    behandlingsårsak: valgtBehandlingsårsak as Behandlingsårsak,
+                    kravMottatt: valgtDato as string,
+                    barn: valgtBarn,
+                };
+                axiosRequest<Ressurs<void>, RevurderingInnhold>({
+                    method: 'POST',
+                    url: `/familie-ef-sak/api/revurdering/${fagsakId}`,
+                    data: revurderingInnhold,
+                })
+                    .then((response) => {
+                        if (response.status === RessursStatus.SUKSESS) {
+                            navigate(`/behandling/${response.data}`);
+                        } else {
+                            settFeilmeldingModal(response.frontendFeilmelding || response.melding);
+                        }
+                    })
+                    .finally(() => {
+                        settSenderInnBehandling(false);
+                    });
+            } else {
+                settFeilmeldingModal('Vennligst fyll ut alle felter');
+            }
+        }
+    };
+
+    return (
+        <UIModalWrapper
+            modal={{
+                tittel: 'Opprett ny behandling',
+                lukkKnapp: true,
+                visModal: visModal,
+                onClose: () => settVisModal(false),
+            }}
+        >
+            <StyledSelect
+                label="Behandlingstype"
+                value={valgtBehandlingstype || ''}
+                onChange={(e) => {
+                    settValgtBehandlingstype(e.target.value as Behandlingstype);
+                    settValgtBehandlingsårsak(undefined);
+                    settFeilmeldingModal(undefined);
+                }}
+            >
+                <option value="">Velg</option>
+                <option value={Behandlingstype.REVURDERING}>Revurdering</option>
+                <option disabled={!visOpprettTilbakekreving} value={Behandlingstype.TILBAKEKREVING}>
```

Comment:
Ta bort hele option heller enn disable?

Reply:
Personlig liker jeg disabled for å skjønne at `valget finnes, men er ikke tilgjengelig`, men her må kanskje de funksjonelle mene noe om dette også?